src/components/inputs/localized-rich-text-input/rich-text-input.js

```diff
@@ -113,15 +113,7 @@ class RichTextInput extends React.PureComponent {
       return (
         <LanguagesControlButton
           isClosed={true}
-          onClick={() => {
-            // expand all multiline language inputs in case the
```

Comment:
neither Filip or Maria could remember why we had this special logic...

Reply:
This logic is also in `<LocalizedMultilineTextInput>`. Should the behaviour remain the same?

Reply:
I'll ask Filip

Reply:
Updated in last commit